src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -208,6 +208,8 @@
 
   "commands.cwe.playerNotFound": "Player not found",
 
+  "commands.cweather.reset": "Reset weather",
```

Comment:
Perhaps something slightly more descriptive, like "stopped overriding the weather"?